grammars/codeblock.json

```diff
@@ -1,6 +1,6 @@
 {
     "fileTypes": [],
-    "injectionSelector": "L:markup.fenced_code.block.markdown",
+    "injectionSelector": "L:markup.fenced_code.block.markdown - meta.embedded.block",
```

Comment:
Instead of injecting our `kotlin-code-block` into `markup.fenced_code.block.markdown`, we should inject it into `text.html.markdown` and then match fenced Kotlin code blocks manually.

Doing that avoids matching any code block containing the string "kotlin". This is also the approach taken by the languages supported by VSCode's official Markdown grammar ([see here](https://github.com/microsoft/vscode/blob/aa82bd59b24ebccde2b38c56ca8050377ecdee1d/extensions/markdown-basics/syntaxes/markdown.tmLanguage.json#L65)) and the recommended approach for third-party extensions ([see here](https://github.com/microsoft/vscode/issues/71888#issuecomment-480961494)).

Here's an example that we could adapt to Kotlin and use as a drop-in replacement for our `codeblock.json`:

https://github.com/mjbvz/vscode-fenced-code-block-grammar-injection-example/blob/master/syntaxes/codeblock.json

Reply:
I agree with you. Fixed.